components/entry.js

```diff
@@ -109,56 +66,112 @@ const Entry = ({ info }) => {
+                        sx={{
+                          color: 'secondary',
+                          mr: [4],
+                          mb: [pad ? 1 : 0, pad ? 1 : 0, 0],
+                          mt: [0, 0, 1],
+                          cursor: 'pointer',
+                          transition: '0.15s',
+                          display: ['block', 'block', 'inline-block'],
+                          float: ['left', 'left', 'initial'],
+                          clear: ['left', 'left', 'initial'],
+                          '&:hover': {
+                            color: 'text',
+                          },
+                        }}
+                      >
+                        {link.label}
+                        <Box as='span' sx={{ position: 'relative' }}>
+                          <Text
+                            as='span'
+                            sx={{
+                              position: 'absolute',
+                              top: '-5px',
+                              left: '3px',
+                              fontSize: [4],
+                            }}
+                          >
+                            ↗
+                          </Text>
+                        </Box>
+                      </Text>
+                    </WrappedLink>
+                  )
+                })}
+              </Box>
+            </Box>
+          </Box>
+          <Box>
             <Box
               sx={{
-                mt: ['10px'],
+                textAlign: 'right',
+                display: ['none', 'none', 'block'],
+                mt: ['-1px'],
               }}
             >
-              {tags.map((tag) => (
-                <Tag
-                  key={tag}
-                  label={tag}
-                  sx={{
-                    mr: [3],
-                    color: 'secondary',
-                    cursor: 'pointer',
-                  }}
-                />
-              ))}
+              {tags
+                .sort((a, b) => (a > b ? 1 : -1))
+                .map((tag) => (
+                  <Tag
+                    key={tag}
+                    label={tag}
+                    sx={{
+                      ml: [2],
+                      mr: [0],
+                      color: 'secondary',
+                    }}
+                  />
+                ))}
             </Box>
+            {icon && (
+              <WrappedLink url={links[0].url}>
+                <Icon icon={icon} color={color} />
+              </WrappedLink>
+            )}
           </Box>
         </Grid>
       </Box>
     </Box>
   )
 }
 
-export default Entry
+function WrappedLink({ url, children }) {
+  if (url.startsWith('/research')) {
+    return (
+      <NextLink href={url} passHref={true}>
+        <Link sx={{ textDecoration: 'none' }}>{children}</Link>
+      </NextLink>
+    )
+  } else {
+    return (
+      <Link sx={{ textDecoration: 'none' }} href={url}>
+        {children}
+      </Link>
```

Comment:
For tracking, I have a slight preference to using a our [`TaggedLink`](https://github.com/carbonplan/research/blob/e7d4cae1dad54aad6958052563cac02e067d4294/components/tagged-link.js#L11) component here:

```js
<TaggedLink action={action} category={type} href={url}>{children}</TaggedLink>
```

where `category` is either `downloads` or `external`

and `action` is either `PDF` or `tool` or `website`